content/posts/2019-06-03-activity-log-using-aws-amplify-and-react/index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Build a Post activity log using React & AWS Amplify
+description: "In this tutorial, we’ll walkthrough on How to create an activity log using AWS Amplify, GraphQL & React."
```

Comment:
In this tutorial, we’ll walkthrough how to create an activity log using AWS Amplify, GraphQL & React.